tests/tests/conftest.py

```diff
@@ -9,11 +9,43 @@ def run_around_tests():
     # Calling yield triggers the test execution
     yield
 
+
 def pytest_sessionfinish(session, exitstatus):
-    # Automatically run after all tests in order to ensure that no kssh-client config files stick around
+    # Automatically run after all tests in order to ensure that no kssh-client
+    # configs stick around
     tc = TestConfig.getDefaultTestConfig()
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.staging/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.prod/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam}.ssh.root_everywhere/kssh-client.config || true" )
-    run_command(f"keybase fs rm /keybase/team/{tc.subteam_secondary}/kssh-client.config || true" )
\ No newline at end of file
+    run_command(
+        f'echo \'{{"method": "del", '
+        f'"params": {{"options": {{"team": '
+        f'"{tc.subteam}.ssh", "namespace": "__sshca", '
+        f'"entryKey": "kssh_config"}}}}}}\' | '
+        f"xargs -0 -I del keybase kvstore api -m del || true"
```

Comment:
eww. :/ suggest pulling this up into a method that builds the full command string from the name of the team. or whatever generally feels more readable and less repeated across all of these.